superset-frontend/src/views/CRUD/welcome/ActivityTable.tsx

```diff
@@ -79,31 +97,68 @@ const ActivityContainer = styled.div`
   }
 `;
 
+const UNTITLED = t('[Untitled]');
+const UNKNOWN_TIME = t('Unknown');
+// translation keys for last action on
+const TRANS_LAST_VIEWED = `Last viewed %s`;
+const TRANS_LAST_MODIFIED = `Last modified %s`;
+
+const getEntityTitle = (e: ActivityObject) => {
```

Comment:
@ktmud  Just one question. Are you sure these are the returns?
As I don't know much about the application, I'm very cautious about typing
If you're sure, I agree

Reply:
It's replicating the same logic as the deleted code. Just with some typing help.